test/CMakeLists.txt

```diff
@@ -46,11 +46,12 @@ endforeach()
 set_target_properties(logging PROPERTIES COMPILE_DEFINITIONS LOGGING_LEVEL_ALL)
 
 add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/test/data/utrecht_tiles/0/003/196.gph
+  COMMAND ${VALHALLA_SOURCE_DIR}/scripts/valhalla_build_timezones 2>/dev/null > test/data/utrecht_tiles/tz.sqlite
```

Comment:
build timezone data so that we can associate it to our tiles. @PureW i know you did something else here. what do you prefer, building the timezone stuff on the fly or checking in the sqlite db?

Reply:
I ended up not having to add any timezone information. But generally, building stuff on the fly is preferred over checking in binary blobs of data. With binary blobs you lose any explainability and chance to easily tweak and make changes.

Reply:
FYI this seems to have broken the build for running mapmatch tests on Mac. I had to install `brew install spatialite-tools` which I think is expected from the CircleCI change above. Even after installing spatialite-tools though I get this error:
```
$ ./scripts/valhalla_build_timezones
spatialite not found which is required.  Please install via:  sudo apt-get install spatialite-bin
```

/cc @yuzheyan who is seeing the same issue.